beetsplug/lastgenre/__init__.py

```diff
@@ -179,12 +198,19 @@ def _resolve_genres(self, tags):
                     parents = [find_parents(tag, self.c14n_branches)[-1]]
 
                 tags_all += parents
-                if len(tags_all) >= count:
+                # Stop if we have enough tags already, unless we need to find
+                # the most specific tag (instead of the most popular).
+                if (not self.config['prefer_specific']
+                        and len(tags_all) >= count):
```

Comment:
Yes; thanks, @Kraymer! I can't really explain why flake8 would prefer that, but OK…